lib/logging/lib/logging.c

```diff
@@ -49,12 +43,16 @@ logging_init(const struct logging_params *const params)
     if (!params)
         return merr(EINVAL);
 
-    if (!params->lp_enabled)
-        return 0;
-
     if (params->lp_level < LOG_EMERG || params->lp_level > LOG_DEBUG)
         return merr(EINVAL);
 
+    if (params->lp_destination < LOG_DEST_STDOUT || params->lp_destination > LOG_DEST_SYSLOG)
+        return merr(EINVAL);
```

Comment:
Don't you have `LOG_DEST_MIN` and `_MAX`?  Alternatively, (1) get rid of `_MIN` and `_MAX` and use `0 <= v < LOG_DEST_COUNT`, or (2) use a switch.

Reply:
oh yeah, I do. Whoops :).